src/reference/asciidoc/smb.adoc

```diff
@@ -0,0 +1,109 @@
+[[smb]]
+== SMB Support
+
+Spring Integration provides support for file transfer operations with SMB.
+
+The https://en.wikipedia.org/wiki/Server_Message_Block[Server Message Block] (SMB) is a simple network protocol that lets you transfer files to a shared file server.
+
+You need to include this dependency into your project:
+
+====
+[source, xml, subs="normal", role="primary"]
+.Maven
+----
+<dependency>
+    <groupId>org.springframework.integration</groupId>
+    <artifactId>spring-integration-smb</artifactId>
+    <version>{project-version}</version>
+</dependency>
+----
+[source, groovy, subs="normal", role="secondary"]
+.Gradle
+----
+compile "org.springframework.integration:spring-integration-smb:{project-version}"
+----
+====
+
+=== Overview
+
+The Java CIFS Client Library has been chosen as a Java implementation for the CIFS/SMB networking protocol.
```

Comment:
I think we need to have a link for that library from here.

Reply:
I thought I put that in there... let me check.